boost_manager/src/activator.rs

```diff
@@ -96,7 +96,7 @@ where
         manifest: RewardManifest,
     ) -> Result<()> {
         // get latest boosted hexes info from mobile config
```

Comment:
```suggestion
        // get active boosted hexes info from mobile config
```

Reply:
I think the update to to the function name renders the comment unnecessary. 
Good looking out.